packages/lighthouse-spa/src/app/dashboard/pages/analysis/analysis.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { DashboardService } from 'app/dashboard/dashboard.service';
+import dayjs from 'dayjs';
```

Comment:
Instead of this use the date pipe

https://angular.io/api/common/DatePipe#usage-notes rest all looks good

Reply:
Done